systers_portal/blog/views.py

```diff
@@ -361,3 +361,28 @@ def get_context_data(self, **kwargs):
                                                  slug=self.kwargs['slug'])
         context['tag_type'] = "tag"
         return context
+
+
+class AddResourceTypeView(LoginRequiredMixin, PermissionRequiredMixin,
+                          CreateView):
+    """Create a new Resource Type"""
+    template_name = "blog/resource_type.html"
```

Comment:
We have a template called `blog/tag_type.html`. Since tag forms and resource forms for adding and editing those objects will looks the same, the template was generalized. It can be used both by tags and by resource types. That is the reason for having a `tag_type` in our context. Therefore please change the template for this view. Moreover I see that the `blog/resource_type.html` is a copy-paste from the original template.